### PR TITLE
Update blogs.yml

### PR DESCRIPTION
**Pull request by Kunal719**
I want to upload my blog for GCI.

### CODE REVIEW

File: _data/blogs.yml [hunk 1]
```diff
@@ -51,3 +51,9 @@
   blog_img:
   article: GCI 2016
   article_link: http://dcongtinh.blogspot.com/2016/12/google-code-in-2016-google-code-in.html
+
```
Comment (m1guelpf) [inline _data/blogs.yml:107]: Please remove trailing whitespace on line 54

**Review by m1guelpf — CHANGES_REQUESTED** [review]
Please make the changed addresed and provide your live gh-pages link.

File: _data/blogs.yml (hunk 1 above)
Comment (Kunal719) [reply _data/blogs.yml:107]: Someone else has taken that line. Now what to do??

**Review by Kunal719 — COMMENTED** [review]
Done

File: _data/blogs.yml (hunk 1 above)
Comment (abishekvashok) [reply _data/blogs.yml:107]: Just remove that space.

**Comment by m1guelpf** [comment]
@Kunal719 You have to fix the merge errors, check contribuiting.md for details.

**Comment by Kunal719** [comment]
Which Space? Can you please highight?

**Comment by m1guelpf** [comment]
@Kunal719 You have to fix the merge errors, you already fixed the space. Check contribuiting.md for help fixing merge

**Comment by Kunal719** [comment]
What is contributing.md ??

**Comment by oanarosca** [comment]
[This](https://github.com/fossasia/gci16.fossasia.org/blob/gh-pages/CONTRIBUTING.md) file.

**Comment by m1guelpf** [comment]
@Kunal719 It is a file with help on contribuiting.
You can access it [here](https://github.com/fossasia/gci16.fossasia.org/blob/gh-pages/CONTRIBUTING.md)

**Comment by Kunal719** [comment]
So what to do with it? I have done the blog, made that "space" work corrected. Now What do you want?

**Comment by oanarosca** [comment]
@Kunal719 please read that file to see how you must solve the merge conflicts. We cannot approve and merge your PR if there are merge conflicts.

**Review by m1guelpf — APPROVED** [review]

**Comment by Kunal719** [comment]
What are those merge conflicts?

**Comment by m1guelpf** [comment]
@Kunal719 It means that your changes conflict with gh-pages changes, because we merged other PR's before yours and things changed.
You must fix that to let us merge.

**Comment by Kunal719** [comment]
Is my pull request approved?

**Comment by m1guelpf** [comment]
@Kunal719 Not yet, you have to fix merge conflicts. I only approved my last review.

**Comment by Kunal719** [comment]
I am not getting how to merge these conflicts.

**Comment by rhemon** [comment]
@Kunal719 have a look into the CONTRIBUTING.md file, it has links to good tutorials on merge conflicts.

**Comment by niccokunzmann** [comment]
capture
For me, there was a button. @Kunal719 Can you help me and others resolve their merge conflicts?

**Pull request merged**